code/game/atoms_movable.dm

```diff
@@ -3,8 +3,6 @@
 	glide_size = 8
 	appearance_flags = TILE_BOUND|PIXEL_SCALE
 
-	///how many times a this movable had movement procs called on it since Moved() was last called
-	var/move_stacks = 0
```

Comment:
@Kylerace Tell me what i have to check for this.

At first glance, look uneeded and nothing went wrong in testing

Reply:
its hard to explain succinctly but moving something with connect_loc_behalf attached to it like slippery items (like banana peels) and doing nested movement calls on them such that they move from A to B to C and Moved(old_loc = B) is called before Moved(old_loc = A) used to reproduce the issue but shouldnt since c_l_b is a component now. i think what i used to reproduce it was using a conveyor belt to move a banana peel onto a hyperspace turf which moves it to another loc, and that should cause a runtime if htis is still a problem